liquid_node/backup.py

```diff
@@ -326,24 +356,96 @@ def restore_collection_es(src, name):
             continue
         es.post(f"/{name}/_open")
         log.info(f"Restore done in {int(time()-t0)}s")
+
     finally:
         es.delete(f"/_snapshot/restore-{name}/snapshot")
         es.delete(f"/_snapshot/restore-{name}")
+
         rm_cmd = (
             f"./liquid dockerexec {SNOOP_ES_ALLOC} "
             f"rm -rf /es_repo/restore-{name} "
         )
         subprocess.check_call(rm_cmd, shell=True)
 
 
+@retry()
+def restore_hypothesis_es(src, name):
```

Comment:
Why the huge duplication? See suggestions above for refactoring `restore_es`:
- add argument for alloc id
- add argument for es url or suffix
- keep two functions: `restore_es` and `backup_es` with the same arguments